api/crypto/frame_crypto_transformer.h

```diff
@@ -417,10 +425,12 @@ class RTC_EXPORT FrameCryptorTransformer
  private:
   void encryptFrame(std::unique_ptr<webrtc::TransformableFrameInterface> frame);
   void decryptFrame(std::unique_ptr<webrtc::TransformableFrameInterface> frame);
+  void onFrameCryptionStateChanged(FrameCryptionState error);
   rtc::Buffer makeIv(uint32_t ssrc, uint32_t timestamp);
   uint8_t getIvSize();
 
  private:
+  TaskQueueBase* const signaling_thread_;
```

Comment:
Is there a reason to use TaskQueueBase* const signaling_thread_; instead of a pointer to rtc::Thread?

Reply:
We only use the task queue to send events, so there is no need for the entire thread object
like this one https://github.com/webrtc-sdk/webrtc/blob/m114_release/pc/peer_connection_message_handler.h#L47

Reply:
oh ok, lg!